gobblin-core/src/main/java/org/apache/gobblin/converter/avro/JsonElementConversionWithAvroSchemaFactory.java

```diff
@@ -210,4 +216,70 @@ public Schema schema() {
       return this.schema;
     }
   }
-}
+
+  public static class UnionConverter extends ComplexConverter {
```

Comment:
Add javadoc to state the assumption made.

Reply:
Addressed